website/src/components/AdoptersSlider/index.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { makeStyles } from "@material-ui/core/styles";
+import Carousel from "../Carousel";
+
+const useStyles = makeStyles((theme) => ({
+  logoCarousel: {
+    marginTop: theme.spacing(6),
+  },
+}));
+
+const AdopterSlider: React.FC = () => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  var logoSliderSettings = {
+    dots: false,
+    autoplay: true,
+    autoplaySpeed: 0,
```

Comment:
Why is it 0?

Reply:
it is given to achive the smooth scroll effect on slider, by giving `autoplaySpeed:0` there wont be any delay on scroll